components/head.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import Container from './container'
+
+const Head = ({children, title, icon}) => (
```

Comment:
This component seems only used in the doc, so it should not be just in `components/head.js`, but probably something like `components/doc/head.js`

Also, `head` is confusing with `next/head`, should probably rename as `PageTitle` (`components/doc/page-title.js`) or something like that

Reply:
This component is only used for documentation at the moment but it is a global component that could be reused by other pages.

It's ok to rename it to `PageTitle`.